hadoop-ozone/dist/src/main/compose/upgrade/compose/ha/docker-compose.yaml

```diff
@@ -59,35 +61,75 @@ x-om:
     - 9872
 
 x-volumes:
+    - &keytabs ../../../_keytabs:/etc/security/keytabs
+    - &krb5conf ./krb5.conf:/etc/krb5.conf
     - &ozone-dir ../../../..:${OZONE_DIR}
     - &transformation ../../../../libexec/transformation.py:/opt/hadoop/libexec/transformation.py
 
 services:
+  kdc:
+    command: ["krb5kdc","-n"]
+    hostname: kdc
+    image: ${OZONE_TESTKRB5_IMAGE}
+    networks:
+      net:
+        ipv4_address: 10.9.0.2
+    volumes:
+      - *keytabs
+      - ../../../..:/opt/hadoop
```

Comment:
`OZONE_DIR` depends on whether we use `ozone-runner` or `ozone` (with binaries) image for Ozone.  This one should not adapt to that change, because `kdc` container always uses the `ozone-testkrb5` image.  So it needs to be hard-coded as `/opt/hadoop`.